Frameworks/UIKit/UIPopoverController.mm

```diff
@@ -14,81 +14,269 @@
                       animated:(BOOL)animated {
-    UNIMPLEMENTED();
+
+    [_popoverControllerInternal presentPopoverFromRect:rect
+                                                inView:view
+                              permittedArrowDirections:arrowDirections
+                                              animated:animated];
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
 */
 - (void)presentPopoverFromBarButtonItem:(UIBarButtonItem*)item
                permittedArrowDirections:(UIPopoverArrowDirection)arrowDirections
                                animated:(BOOL)animated {
-    UNIMPLEMENTED();
+
+    [_popoverControllerInternal presentPopoverFromBarButtonItem:item
+                                       permittedArrowDirections:arrowDirections
+                                                       animated:animated];
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
 */
 - (void)dismissPopoverAnimated:(BOOL)animated {
-    UNIMPLEMENTED();
+    [_popoverControllerInternal dismissPopoverAnimated:animated];
+}
+
+- (void)_dismissPopoverAnimated:(BOOL)animated completion:(dispatch_block_t)completion {
+    [_popoverControllerInternal dismissPopoverAnimated:animated completion:completion];
+}
+
+/**
+ @Status Interoperable
+*/
+- (void)setDelegate:(id<UIPopoverControllerDelegate>)delegate {
+    [_delegateInternal setDelegate:delegate];
+}
+
+/**
+ @Status Interoperable
+*/
+- (id<UIPopoverControllerDelegate>)delegate {
+    return [_delegateInternal delegate];
 }
 
 /**
  @Status Stub
- @Notes
 */
 - (void)setContentViewController:(UIViewController*)viewController animated:(BOOL)animated {
     UNIMPLEMENTED();
 }
 
 /**
  @Status Stub
- @Notes
+*/
+- (void)setContentViewController:(UIViewController*)viewController {
+    UNIMPLEMENTED();
+}
+
+/**
+ @Status Interoperable
+*/
+- (UIViewController*)contentViewController {
+    return [_popoverControllerInternal contentViewController];
+}
+
+/**
+ @Status Interoperable
+*/
+- (CGSize)popoverContentSize {
+    return [_popoverControllerInternal popoverContentSize];
+}
+
+/**
+ @Status Interoperable
+*/
+- (void)setPopoverContentSize:(CGSize)size {
+    [_popoverControllerInternal setPopoverContentSize:size];
+}
+
+/**
+ @Status Interoperable
 */
 - (void)setPopoverContentSize:(CGSize)size animated:(BOOL)animated {
+    [_popoverControllerInternal setPopoverContentSize:size];
+}
+
+/**
+ @Status Interoperable
+*/
+- (BOOL)isPopoverVisible {
+    return [_popoverControllerInternal isPopoverVisible];
+}
+
+/**
+ @Status Interoperable
+*/
+- (NSArray*)passthroughViews {
+    return [_popoverControllerInternal passthroughViews];
+}
+
+/**
+ @Status Interoperable
+*/
+- (void)setPassthroughViews:(NSArray*)passthroughViews {
+    return [_popoverControllerInternal setPassthroughViews:passthroughViews];
+}
+
+/**
+ @Status Interoperable
+*/
+- (UIPopoverArrowDirection)popoverArrowDirection {
     UNIMPLEMENTED();
```

Comment:
ah...then please update the annotation to @Status Stub

---
In reply to: [62900503](https://github.com/Microsoft/WinObjC/pull/462#discussion_r62900503) [](ancestors = 62900503)